wagtailio/static/sass/components/_sign-up-form.scss

```diff
@@ -0,0 +1,132 @@
+.sign-up-form {
+    $root: &;
+    @include sf-spacing();
+
+    &__heading {
+        // normalise the h2 element
+        margin: 0;
+    }
+
+    &__sub-heading {
+        padding: 10px 0 0 0;
+
+    }
+
+    &__label {
+        display: none;
+    }
```

Comment:
Suggestion: delete this once the screen reader class has been added to the label